### DISCUSSION TITLE
Any way to install Preline without `npm`?

### DISCUSSION THREAD

**Discussion by alexandremjacques** [htmlstreamofficial/preline#51] [Q&A]
I have a Django app that is an old school server rendered app. I would prefer not to add extra step(s) to my build/deploy pipeline to add `npm`.

Is there a way to use Preline without `npm`?

TIA

**Comment by jahaganiev** [comment]
Hey @alexandremjacques! Yes, definitely - you may just download from Github and follow number 2 and 3 steps from the docs page https://preline.co/docs/index.html

Cheers!

**Reply by alexandremjacques** [reply]
Apparently it doesn't work. If I try to run `tailwindcss` CLI, it spits out an error saying:

```
tailwindcss -i ./static/css/tailwind.css -o ./static/css/styles.css

Error: Cannot find module 'preline/plugin'
```

Do I need to import something in `tailwind.config.js`?

TIA

**Reply by jahaganiev** [reply]
Looks like it cannot find the 'preline/plugin', you may add 'path_to/preline/plugin' in the tailwind.config.js file and try out again.

Hope that helps!

Cheers!

**Comment by rodrigoantas** [comment]
this is an old discussion, but I came across this post and found a solution.
you can just paste this in your script:

https://www.jsdelivr.com/package/npm/preline

**Reply by honeyBeexO** [reply]
Thanks man, this really helped me. I can't fathom why they did not mention their cdn version in the docs

**Reply by jahaganiev** [reply]
Please note, Preline doesn’t support CDNs for development. CDNs can be used for production.
